schemas/bill_summaries.py

```diff
@@ -0,0 +1,26 @@
+from pydantic import BaseModel, Field
+
+
+class BillSummaryUsSectionSchema(BaseModel):
+    """'bill_summary_us' section schema."""
+
+    text: str | None = Field(None)
+    id: str | None = Field(None)
+    header: str | None = Field(None)
+
+
+class BillSummaryUsItemSchema(BaseModel):
+    """'bill_summary_us' single item schema."""
+
+    id: str = Field()
```

Comment:
is ` = Field()` required here? 

I guess `id: str` should be enough

Reply:
Field a nice thing to have, we might add descriptions to the field for the schema later on